Build failure on Linux

Compiler version:
```
Using built-in specs.
COLLECT_GCC=gcc-4.9
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/lto-wrapper
Target: x86_64-pc-linux-gnu
Configured with: /builder/Packages/gcc49/src/gcc-4.9.4/configure --prefix=/usr --build=x86_64-pc-linux-gnu --libdir=/usr/lib --libexecdir=/usr/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=https://bugs.archlinux.org/ --enable-languages=c,c++,fortran,go,lto,objc,obj-c++ --enable-shared --enable-threads=posix --with-system-zlib --enable-__cxa_atexit --disable-libunwind-exceptions --enable-clocale=gnu --disable-libstdcxx-pch --disable-libssp --enable-gnu-unique-object --enable-linker-build-id --enable-cloog-backend=isl --enable-lto --enable-plugin --enable-install-libiberty --with-linker-hash-style=gnu --disable-multilib --disable-werror --program-suffix=-4.9 --enable-version-specific-runtime-libs --enable-checking=release
Thread model: posix
gcc version 4.9.4 (GCC) 
```

Error message:
```
 1:02.03 In file included from /builder/Packages/palemoon-git/src/pmbuild/dom/media/mediasource/Unified_cpp_media_mediasource0.cpp:2:0:
 1:02.03 /builder/Packages/palemoon-git/src/Pale-Moon/dom/media/mediasource/ContainerParser.cpp: In constructor ‘mozilla::MP4ContainerParser::AtomParser::AtomParser(const nsACString_internal&, const mozilla::MediaLargeByteBuffer*)’:
 1:02.03 /builder/Packages/palemoon-git/src/Pale-Moon/dom/media/mediasource/ContainerParser.cpp:317:31: error: expected primary-expression before ‘,’ token
 1:02.03          MSE_DEBUGV(AtomParser ,"Checking atom:'%c%c%c%c'",
 1:02.03                                ^
 1:02.03 /builder/Packages/palemoon-git/src/Pale-Moon/dom/media/mediasource/ContainerParser.cpp:318:58: error: ‘MSE_DEBUGV’ was not declared in this scope
 1:02.03                     typec[0], typec[1], typec[2], typec[3]);
 1:02.03                                                           ^
 1:06.16 In file included from /builder/Packages/palemoon-git/src/pmbuild/dom/media/mediasource/Unified_cpp_media_mediasource0.cpp:110:0:
 1:06.16 /builder/Packages/palemoon-git/src/Pale-Moon/dom/media/mediasource/TrackBuffersManager.cpp: At global scope:
 1:06.16 Warning: -Wunused-function in /builder/Packages/palemoon-git/src/Pale-Moon/dom/media/mediasource/TrackBuffersManager.cpp: ‘const char* mozilla::AppendStateToStr(mozilla::SourceBufferContentManager::AppendState)’ defined but not used
 1:06.16 /builder/Packages/palemoon-git/src/Pale-Moon/dom/media/mediasource/TrackBuffersManager.cpp:26:1: warning: ‘const char* mozilla::AppendStateToStr(mozilla::SourceBufferContentManager::AppendState)’ defined but not used [-Wunused-function]
 1:06.16  AppendStateToStr(TrackBuffersManager::AppendState aState)
 1:06.16  ^
 1:06.20 In the directory  /builder/Packages/palemoon-git/src/pmbuild/dom/media/mediasource
 1:06.20 The following command failed to execute properly:
 1:06.20 g++-4.9 -o Unified_cpp_media_mediasource0.o -c -I../../../dist/stl_wrappers -I../../../dist/system_wrappers -include /builder/Packages/palemoon-git/src/Pale-Moon/config/gcc_hidden.h -DSTATIC_EXPORTABLE_JS_API -DMOZILLA_INTERNAL_API -DIMPL_LIBXUL -DMOZ_GLUE_IN_PROGRAM -DAB_CD=en-US -DNO_NSPR_10_SUPPORT -I/builder/Packages/palemoon-git/src/Pale-Moon/dom/media/mediasource -I. -I../../../dist/include -I/usr/include/nspr -I/usr/include/nss -I/usr/include/pixman-1 -fPIC -D_FORTIFY_SOURCE=2 -O2 -DMOZILLA_CLIENT -include ../../../mozilla-config.h -MD -MP -MF .deps/Unified_cpp_media_mediasource0.o.pp -D_FORTIFY_SOURCE=2 -O2 -Wall -Wempty-body -Woverloaded-virtual -Wsign-compare -Wwrite-strings -Wno-invalid-offsetof -Wcast-align -march=native -mtune=native -O2 -pipe -fstack-protector-strong -fno-exceptions -fno-strict-aliasing -fno-rtti -ffunction-sections -fdata-sections -fno-exceptions -fno-math-errno -std=gnu++0x -pthread -pipe -DNDEBUG -DTRIMMED -O2 -march=native -mtune=native -fomit-frame-pointer /builder/Packages/palemoon-git/src/pmbuild/dom/media/mediasource/Unified_cpp_media_mediasource0.cpp
```

--- a/dom/media/mediasource/ContainerParser.cpp
+++ b/dom/media/mediasource/ContainerParser.cpp
@@ -32,6 +32,7 @@ extern PRLogModuleInfo* GetMediaSourceLog();
 #define MSE_DEBUGV(name, arg, ...) PR_LOG(GetMediaSourceLog(), PR_LOG_DEBUG + 1, (TOSTRING(name) "(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
 #else
 #define MSE_DEBUG(...)
+#define MSE_DEBUGV(...)
 #endif
 
 namespace mozilla {

--- a/dom/media/mediasource/ContainerParser.cpp
+++ b/dom/media/mediasource/ContainerParser.cpp
@@ -29,7 +29,7 @@ extern PRLogModuleInfo* GetMediaSourceLog();
 #define STRINGIFY(x) #x
 #define TOSTRING(x) STRINGIFY(x)
 #define MSE_DEBUG(name, arg, ...) PR_LOG(GetMediaSourceLog(), PR_LOG_DEBUG, (TOSTRING(name) "(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
-#define MSE_DEBUGV(name, arg, ...) PR_LOG(GetMediaSourceLog(), PR_LOG_DEBUG, (TOSTRING(name) "(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
+#define MSE_DEBUGV(name, arg, ...) PR_LOG(GetMediaSourceLog(), PR_LOG_DEBUG + 1, (TOSTRING(name) "(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
 #else
 #define MSE_DEBUG(...)
 #endif

--- a/dom/media/mediasource/SourceBufferResource.cpp
+++ b/dom/media/mediasource/SourceBufferResource.cpp
@@ -24,7 +24,7 @@ PRLogModuleInfo* GetSourceBufferResourceLog()
 }
 
 #define SBR_DEBUG(arg, ...) PR_LOG(GetSourceBufferResourceLog(), PR_LOG_DEBUG, ("SourceBufferResource(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
-#define SBR_DEBUGV(arg, ...) PR_LOG(GetSourceBufferResourceLog(), PR_LOG_DEBUG+1, ("SourceBufferResource(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
+#define SBR_DEBUGV(arg, ...) PR_LOG(GetSourceBufferResourceLog(), PR_LOG_DEBUG + 1, ("SourceBufferResource(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
 #else
 #define SBR_DEBUG(...)
 #define SBR_DEBUGV(...)

--- a/dom/media/mediasource/TrackBuffersManager.cpp
+++ b/dom/media/mediasource/TrackBuffersManager.cpp
@@ -15,10 +15,15 @@
 
 #include <limits>
 
+#ifdef PR_LOGGING
 extern PRLogModuleInfo* GetMediaSourceLog();
 
 #define MSE_DEBUG(arg, ...) PR_LOG(GetMediaSourceLog(), PR_LOG_DEBUG, ("TrackBuffersManager(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
-#define MSE_DEBUGV(arg, ...) PR_LOG(GetMediaSourceLog(), PR_LOG_DEBUG, ("TrackBuffersManager(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
+#define MSE_DEBUGV(arg, ...) PR_LOG(GetMediaSourceLog(), PR_LOG_DEBUG + 1, ("TrackBuffersManager(%p:%s)::%s: " arg, this, mType.get(), __func__, ##__VA_ARGS__))
+#else
+#define MSE_DEBUG(...)
+#define MSE_DEBUGV(...)
+#endif
 
 namespace mozilla {
 

